### ISSUE TITLE
Possible to add KML support?

### ISSUE THREAD

**Issue by ThomasDaheim** [xbgmsharp/piwigo-openstreetmap#70]
Hi,

since my track info is usually in KML I have hacked around to add KML support to the plugin. But I'm sure thats not the proper way of doing things :-) Any chance, that KML support might gets added to the plugin in the near future?

Thank!
Thomas

**Comment by xbgmsharp** [comment]
I will look and see if there is a KML plugin for Leaflet.

**Comment by ThomasDaheim** [comment]
Yes, there is. And thats how I added KML support :-) Simple adding kml as extension into gpx.inc.php and have a separate osm-kml.tpl. But as I said, its just a hack and nothing to upload here :-)

**Comment by xbgmsharp** [comment]
Could you submit you code via pull request?
It would be a good start.

**Comment by ThomasDaheim** [comment]
Sorry, I guess my non-existing knowledge of git process doesn't help here.

I have created a local branch "kml-support" and added my changes but my pull request fails "It seems you don't have permission to push your changes to this repository"...

**Comment by xbgmsharp** [comment]
You have to pull the change to your branch.
Then you can make a pull request to my repository.

**Comment by ThomasDaheim** [comment]
Sorry, I always get "you don't have permission" - even when trying to Publish my commited local changes...

**Comment by xbgmsharp** [comment]
1. you fork the repo
2. you clone your new repo
3. you apply your change
4. you push your change to you repo.
5. via the web UI you can create a pull request

**Comment by ThomasDaheim** [comment]
Thanks, seems to have worked without errors :-)
